core/src/main/java/org/frankframework/configuration/digester/ValidateAttributeRule.java

```diff
@@ -161,10 +173,25 @@ private void checkDeprecationAndConfigurationWarning(String name, Method m) {
 			}
 
 			if (isDeprecated) {
-				addSuppressableWarning(msg, SuppressKeys.DEPRECATION_SUPPRESS_KEY);
+				addSuppressibleWarning(msg, SuppressKeys.DEPRECATION_SUPPRESS_KEY);
 			} else {
 				addLocalWarning(msg);
 			}
+			return;
+		}
+		// Check enum Configuration Warnings
+		if (!m.getParameters()[0].getType().isEnum())
+			return; // Skip non-enum setters
+		try {
+			Object o = ClassUtils.parseValueToSet(m, value);
```

Comment:
Kan je hier niet beter EnumUtils gebruiken?

Reply:
Ik zie niet zo snel hoe dit ook met de EnumUtils kan. Die hebben vaak net een ander doel, dacht ik. Daarnaast wilde ik geen extra verwerking van velden als het geen enum is, dus vandaar de check op 183 en de snelle return.